db/src/models/Tag.js

```diff
@@ -0,0 +1,41 @@
+import { Model } from 'objection';
+import Release from './Release.js';
+
+export default class Tag extends Model {
+  static tableName = 'tags';
+  static idColumn = 'id';
+  static jsonSchema = {
+    type: 'object',
+    required: ['value'],
+    properties: {
+      value: { type: 'string' },
+    },
+  }
+
+  static findOrCreate = async (value) => {
+    let tag = await Tag.query().where('value', value).first();
+    if (!tag) {
+      tag = await Tag.query().insert({ value });
```

Comment:
will there ever be conflicts if a record already exists? if so, we should add logic to handle if the record already exists for any unforeseen reasons.

Reply:
i believe this code should handle that.  if a tag exists it returns it, otherwise it inserts it and returns it. do you think there should be something more?

Reply:
nope this is good